releases/1.6/edge/kubeflow/bundle.yaml

```diff
@@ -0,0 +1,200 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook:
+    charm: admission-webhook
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: admission-webhook-operator
+  argo-controller:
+    charm: argo-controller
+    channel: 3.2/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  argo-server:
+    charm: argo-server
+    channel: 3.3/edge
+    scale: 1
+    _github_repo_name: argo-operators
+  dex-auth:
+    charm: dex-auth
+    channel: 2.31/edge
+    scale: 1
+    trust: true
+    _github_repo_name: dex-auth-operator
+  envoy:
+    charm: envoy
+    channel: 1.20/edge
+    scale: 1
+    _github_repo_name: envoy-operator
+  istio-ingressgateway:
+    charm: istio-gateway
+    channel: 1.11/edge
+    scale: 1
+    trust: true
+    _github_repo_name: istio-operators
+  istio-pilot:
+    charm: istio-pilot
+    channel: 1.11/edge
```

Comment:
Isn't our istio 1.11/edge tied to latest/edge in this case?

Reply:
yeah I think 1.11/edge really means "whatever we eventually put in 1.11/edge",   since 1.11 is the version we're actively working on now.  Not necessarily what is there now